core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -274,9 +282,15 @@ sealed trait ZValidation[+W, +E, +A] { self =>
 object ZValidation extends LowPriorityValidationImplicits {
 
   final case class Failure[+W, +E](log: Chunk[W], errors: NonEmptyChunk[E]) extends ZValidation[W, E, Nothing] {
+
+    /** The errors represented in a way which discards the order in which they occurred. */
     lazy val errorsUnordered: NonEmptyMultiSet[E] = NonEmptyMultiSet.fromIterable(errors.head, errors.tail)
+
+    /** The description of the failure. */
+    lazy val message: String =
```

Comment:
Isn't this covered by `Debug` instance?

Reply:
I think the current `Debug` instance is all-right as it is, but `message` is formatted with newlines and so on to enhance readability, which I think is very useful for many purposes.